catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/RealTimeGetDelegate.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.source.solr;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.filter.impl.SimpleFilterDelegate;
+import java.util.List;
+
+public class RealTimeGetDelegate extends SimpleFilterDelegate<Boolean> {
+
+  @Override
+  public <S> Boolean defaultOperation(
+      Object property, S literal, Class<S> literalClass, Enum operation) {
+    return false;
+  }
+
+  @Override
+  public Boolean and(List<Boolean> operands) {
+    return operands.contains(Boolean.TRUE);
```

Comment:
❓Shouldn't the implementations of `and()` and `or()` be swapped?

Reply:
Not for our use case, we're going to use whatever boolean is returned by this filter delegate to determine whether we should do a real time get or not. We only want to do a real time get on queries for metacards with specific ids. Some more info can be found here https://lucene.apache.org/solr/guide/7_4/realtime-get.html but I think the best way to understand is with examples so here are some: 
id = 123 -> should return true
(id =123) AND (name = foo) -> true
(id = 123) OR ((ID = 456) AND (name = foo)) -> true
(id = 123) OR (name = foo) -> false
NOT (id =123) -> false